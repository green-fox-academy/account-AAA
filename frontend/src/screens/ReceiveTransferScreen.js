import React from 'react';
import QRCode from 'react-native-qrcode-svg';
import { Text, View } from 'native-base';
import PropTypes from 'prop-types';
import navigationPropTypes from '../helpers/navigationPropTypes';
import styles from '../styles/ReceiveTransferScreenStyle';

export default function ReceiveTransferScreen({ userId, navigation }) {
  const { id, depositName } = navigation.getParam('account');
  const codeData = {
    depositId: id,
    userId,
  };
  return (
    <View style={styles.receiveScreenView}>
      <Text style={styles.depositName}>{depositName}</Text>
      <QRCode
        value={JSON.stringify(codeData)}
        size={300}
      />
    </View>
  );
}

ReceiveTransferScreen.propTypes = {
  userId: PropTypes.number.isRequired,
  navigation: navigationPropTypes.isRequired,
};
