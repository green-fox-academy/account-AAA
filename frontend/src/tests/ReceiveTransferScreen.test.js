import React from 'react';
import renderer from 'react-test-renderer';
import ReceiveTransferScreen from '../screens/ReceiveTransferScreen';

describe('<ReceiveTransferScreen />', () => {
  it('render correctly', () => {
    const data = {
      account: {
        id: 1,
        depositName: 'Main Account',
      },
    };
    const navigation = {
      getParam: (key) => data[key],
    };
    const tree = renderer
      .create(
        <ReceiveTransferScreen navigation={navigation} userId={123} />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
