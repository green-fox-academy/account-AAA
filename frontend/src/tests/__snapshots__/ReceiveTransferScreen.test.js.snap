// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReceiveTransferScreen /> render correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "marginTop": 50,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        },
        Object {
          "fontSize": 25,
          "marginBottom": 50,
        },
      ]
    }
    uppercase={false}
  >
    Main Account
  </Text>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight={300}
    bbWidth={300}
    height={300}
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        undefined,
        Object {
          "opacity": 1,
        },
        Object {
          "flex": 0,
          "height": 300,
          "width": 300,
        },
      ]
    }
    vbHeight={300}
    vbWidth={300}
    width={300}
  >
    <RNSVGGroup
      fill={
        Array [
          0,
          4278190080,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      font={Object {}}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={Array []}
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    >
      <RNSVGDefs>
        <RNSVGLinearGradient
          gradient={
            Array [
              0,
              -65536,
              1,
              -16711681,
            ]
          }
          gradientTransform={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          gradientUnits={0}
          name="grad"
          x1="0%"
          x2="100%"
          y1="0%"
          y2="100%"
        />
      </RNSVGDefs>
      <RNSVGGroup
        fill={
          Array [
            0,
            4278190080,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        font={Object {}}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        opacity={1}
        originX={0}
        originY={0}
        propList={Array []}
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      >
        <RNSVGRect
          fill={
            Array [
              0,
              4294967295,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          height={300}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          rx={0}
          ry={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth={1}
          vectorEffect={0}
          width={300}
          x={0}
          y={0}
        />
      </RNSVGGroup>
      <RNSVGGroup
        fill={
          Array [
            0,
            4278190080,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        font={Object {}}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        opacity={1}
        originX={0}
        originY={0}
        propList={Array []}
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      >
        <RNSVGPath
          d="M0 5.172413793103448 L72.41379310344828 5.172413793103448 M82.75862068965517 5.172413793103448 L93.10344827586206 5.172413793103448 M103.44827586206897 5.172413793103448 L124.13793103448276 5.172413793103448 M134.48275862068965 5.172413793103448 L165.51724137931035 5.172413793103448 M186.20689655172413 5.172413793103448 L206.89655172413794 5.172413793103448 M227.58620689655174 5.172413793103448 L300 5.172413793103448 M0 15.517241379310345 L10.344827586206897 15.517241379310345 M62.06896551724138 15.517241379310345 L72.41379310344828 15.517241379310345 M82.75862068965517 15.517241379310345 L93.10344827586206 15.517241379310345 M103.44827586206897 15.517241379310345 L113.79310344827587 15.517241379310345 M134.48275862068965 15.517241379310345 L144.82758620689657 15.517241379310345 M165.51724137931035 15.517241379310345 L175.86206896551724 15.517241379310345 M206.89655172413794 15.517241379310345 L217.24137931034483 15.517241379310345 M227.58620689655174 15.517241379310345 L237.93103448275863 15.517241379310345 M289.65517241379314 15.517241379310345 L300 15.517241379310345 M0 25.862068965517242 L10.344827586206897 25.862068965517242 M20.689655172413794 25.862068965517242 L51.724137931034484 25.862068965517242 M62.06896551724138 25.862068965517242 L72.41379310344828 25.862068965517242 M93.10344827586206 25.862068965517242 L134.48275862068965 25.862068965517242 M165.51724137931035 25.862068965517242 L175.86206896551724 25.862068965517242 M196.55172413793105 25.862068965517242 L217.24137931034483 25.862068965517242 M227.58620689655174 25.862068965517242 L237.93103448275863 25.862068965517242 M248.27586206896552 25.862068965517242 L279.3103448275862 25.862068965517242 M289.65517241379314 25.862068965517242 L300 25.862068965517242 M0 36.20689655172414 L10.344827586206897 36.20689655172414 M20.689655172413794 36.20689655172414 L51.724137931034484 36.20689655172414 M62.06896551724138 36.20689655172414 L72.41379310344828 36.20689655172414 M82.75862068965517 36.20689655172414 L93.10344827586206 36.20689655172414 M103.44827586206897 36.20689655172414 L134.48275862068965 36.20689655172414 M144.82758620689657 36.20689655172414 L165.51724137931035 36.20689655172414 M175.86206896551724 36.20689655172414 L196.55172413793105 36.20689655172414 M227.58620689655174 36.20689655172414 L237.93103448275863 36.20689655172414 M248.27586206896552 36.20689655172414 L279.3103448275862 36.20689655172414 M289.65517241379314 36.20689655172414 L300 36.20689655172414 M0 46.55172413793103 L10.344827586206897 46.55172413793103 M20.689655172413794 46.55172413793103 L51.724137931034484 46.55172413793103 M62.06896551724138 46.55172413793103 L72.41379310344828 46.55172413793103 M103.44827586206897 46.55172413793103 L113.79310344827587 46.55172413793103 M124.13793103448276 46.55172413793103 L134.48275862068965 46.55172413793103 M144.82758620689657 46.55172413793103 L155.17241379310346 46.55172413793103 M165.51724137931035 46.55172413793103 L175.86206896551724 46.55172413793103 M186.20689655172413 46.55172413793103 L206.89655172413794 46.55172413793103 M227.58620689655174 46.55172413793103 L237.93103448275863 46.55172413793103 M248.27586206896552 46.55172413793103 L279.3103448275862 46.55172413793103 M289.65517241379314 46.55172413793103 L300 46.55172413793103 M0 56.896551724137936 L10.344827586206897 56.896551724137936 M62.06896551724138 56.896551724137936 L72.41379310344828 56.896551724137936 M93.10344827586206 56.896551724137936 L103.44827586206897 56.896551724137936 M113.79310344827587 56.896551724137936 L124.13793103448276 56.896551724137936 M134.48275862068965 56.896551724137936 L144.82758620689657 56.896551724137936 M155.17241379310346 56.896551724137936 L186.20689655172413 56.896551724137936 M196.55172413793105 56.896551724137936 L217.24137931034483 56.896551724137936 M227.58620689655174 56.896551724137936 L237.93103448275863 56.896551724137936 M289.65517241379314 56.896551724137936 L300 56.896551724137936 M0 67.24137931034483 L72.41379310344828 67.24137931034483 M82.75862068965517 67.24137931034483 L93.10344827586206 67.24137931034483 M103.44827586206897 67.24137931034483 L113.79310344827587 67.24137931034483 M124.13793103448276 67.24137931034483 L134.48275862068965 67.24137931034483 M144.82758620689657 67.24137931034483 L155.17241379310346 67.24137931034483 M165.51724137931035 67.24137931034483 L175.86206896551724 67.24137931034483 M186.20689655172413 67.24137931034483 L196.55172413793105 67.24137931034483 M206.89655172413794 67.24137931034483 L217.24137931034483 67.24137931034483 M227.58620689655174 67.24137931034483 L300 67.24137931034483 M82.75862068965517 77.58620689655173 L93.10344827586206 77.58620689655173 M103.44827586206897 77.58620689655173 L134.48275862068965 77.58620689655173 M155.17241379310346 77.58620689655173 L186.20689655172413 77.58620689655173 M206.89655172413794 77.58620689655173 L217.24137931034483 77.58620689655173 M0 87.93103448275862 L10.344827586206897 87.93103448275862 M20.689655172413794 87.93103448275862 L41.37931034482759 87.93103448275862 M51.724137931034484 87.93103448275862 L82.75862068965517 87.93103448275862 M113.79310344827587 87.93103448275862 L124.13793103448276 87.93103448275862 M134.48275862068965 87.93103448275862 L144.82758620689657 87.93103448275862 M155.17241379310346 87.93103448275862 L165.51724137931035 87.93103448275862 M175.86206896551724 87.93103448275862 L186.20689655172413 87.93103448275862 M227.58620689655174 87.93103448275862 L237.93103448275863 87.93103448275862 M258.62068965517244 87.93103448275862 L268.9655172413793 87.93103448275862 M279.3103448275862 87.93103448275862 L300 87.93103448275862 M41.37931034482759 98.27586206896551 L62.06896551724138 98.27586206896551 M72.41379310344828 98.27586206896551 L82.75862068965517 98.27586206896551 M93.10344827586206 98.27586206896551 L103.44827586206897 98.27586206896551 M113.79310344827587 98.27586206896551 L124.13793103448276 98.27586206896551 M134.48275862068965 98.27586206896551 L165.51724137931035 98.27586206896551 M206.89655172413794 98.27586206896551 L217.24137931034483 98.27586206896551 M227.58620689655174 98.27586206896551 L237.93103448275863 98.27586206896551 M248.27586206896552 98.27586206896551 L268.9655172413793 98.27586206896551 M279.3103448275862 98.27586206896551 L300 98.27586206896551 M0 108.62068965517241 L20.689655172413794 108.62068965517241 M31.03448275862069 108.62068965517241 L41.37931034482759 108.62068965517241 M62.06896551724138 108.62068965517241 L72.41379310344828 108.62068965517241 M82.75862068965517 108.62068965517241 L93.10344827586206 108.62068965517241 M124.13793103448276 108.62068965517241 L144.82758620689657 108.62068965517241 M165.51724137931035 108.62068965517241 L186.20689655172413 108.62068965517241 M206.89655172413794 108.62068965517241 L217.24137931034483 108.62068965517241 M227.58620689655174 108.62068965517241 L289.65517241379314 108.62068965517241 M41.37931034482759 118.96551724137932 L51.724137931034484 118.96551724137932 M82.75862068965517 118.96551724137932 L93.10344827586206 118.96551724137932 M103.44827586206897 118.96551724137932 L113.79310344827587 118.96551724137932 M165.51724137931035 118.96551724137932 L175.86206896551724 118.96551724137932 M186.20689655172413 118.96551724137932 L196.55172413793105 118.96551724137932 M289.65517241379314 118.96551724137932 L300 118.96551724137932 M0 129.31034482758622 L20.689655172413794 129.31034482758622 M31.03448275862069 129.31034482758622 L41.37931034482759 129.31034482758622 M51.724137931034484 129.31034482758622 L82.75862068965517 129.31034482758622 M93.10344827586206 129.31034482758622 L103.44827586206897 129.31034482758622 M124.13793103448276 129.31034482758622 L134.48275862068965 129.31034482758622 M144.82758620689657 129.31034482758622 L165.51724137931035 129.31034482758622 M175.86206896551724 129.31034482758622 L196.55172413793105 129.31034482758622 M206.89655172413794 129.31034482758622 L227.58620689655174 129.31034482758622 M258.62068965517244 129.31034482758622 L279.3103448275862 129.31034482758622 M20.689655172413794 139.6551724137931 L62.06896551724138 139.6551724137931 M82.75862068965517 139.6551724137931 L93.10344827586206 139.6551724137931 M113.79310344827587 139.6551724137931 L124.13793103448276 139.6551724137931 M144.82758620689657 139.6551724137931 L155.17241379310346 139.6551724137931 M165.51724137931035 139.6551724137931 L206.89655172413794 139.6551724137931 M217.24137931034483 139.6551724137931 L248.27586206896552 139.6551724137931 M268.9655172413793 139.6551724137931 L279.3103448275862 139.6551724137931 M289.65517241379314 139.6551724137931 L300 139.6551724137931 M10.344827586206897 150.00000000000003 L31.03448275862069 150.00000000000003 M41.37931034482759 150.00000000000003 L51.724137931034484 150.00000000000003 M62.06896551724138 150.00000000000003 L82.75862068965517 150.00000000000003 M93.10344827586206 150.00000000000003 L144.82758620689657 150.00000000000003 M155.17241379310346 150.00000000000003 L186.20689655172413 150.00000000000003 M196.55172413793105 150.00000000000003 L217.24137931034483 150.00000000000003 M227.58620689655174 150.00000000000003 L237.93103448275863 150.00000000000003 M258.62068965517244 150.00000000000003 L268.9655172413793 150.00000000000003 M279.3103448275862 150.00000000000003 L300 150.00000000000003 M51.724137931034484 160.34482758620692 L62.06896551724138 160.34482758620692 M72.41379310344828 160.34482758620692 L134.48275862068965 160.34482758620692 M144.82758620689657 160.34482758620692 L155.17241379310346 160.34482758620692 M165.51724137931035 160.34482758620692 L175.86206896551724 160.34482758620692 M186.20689655172413 160.34482758620692 L196.55172413793105 160.34482758620692 M206.89655172413794 160.34482758620692 L217.24137931034483 160.34482758620692 M237.93103448275863 160.34482758620692 L268.9655172413793 160.34482758620692 M279.3103448275862 160.34482758620692 L300 160.34482758620692 M10.344827586206897 170.6896551724138 L20.689655172413794 170.6896551724138 M31.03448275862069 170.6896551724138 L41.37931034482759 170.6896551724138 M51.724137931034484 170.6896551724138 L72.41379310344828 170.6896551724138 M82.75862068965517 170.6896551724138 L93.10344827586206 170.6896551724138 M103.44827586206897 170.6896551724138 L113.79310344827587 170.6896551724138 M144.82758620689657 170.6896551724138 L175.86206896551724 170.6896551724138 M206.89655172413794 170.6896551724138 L217.24137931034483 170.6896551724138 M237.93103448275863 170.6896551724138 L258.62068965517244 170.6896551724138 M41.37931034482759 181.0344827586207 L51.724137931034484 181.0344827586207 M72.41379310344828 181.0344827586207 L103.44827586206897 181.0344827586207 M113.79310344827587 181.0344827586207 L134.48275862068965 181.0344827586207 M144.82758620689657 181.0344827586207 L155.17241379310346 181.0344827586207 M165.51724137931035 181.0344827586207 L175.86206896551724 181.0344827586207 M186.20689655172413 181.0344827586207 L196.55172413793105 181.0344827586207 M206.89655172413794 181.0344827586207 L217.24137931034483 181.0344827586207 M237.93103448275863 181.0344827586207 L248.27586206896552 181.0344827586207 M268.9655172413793 181.0344827586207 L279.3103448275862 181.0344827586207 M0 191.3793103448276 L10.344827586206897 191.3793103448276 M51.724137931034484 191.3793103448276 L93.10344827586206 191.3793103448276 M113.79310344827587 191.3793103448276 L196.55172413793105 191.3793103448276 M206.89655172413794 191.3793103448276 L217.24137931034483 191.3793103448276 M227.58620689655174 191.3793103448276 L289.65517241379314 191.3793103448276 M20.689655172413794 201.7241379310345 L31.03448275862069 201.7241379310345 M41.37931034482759 201.7241379310345 L51.724137931034484 201.7241379310345 M103.44827586206897 201.7241379310345 L155.17241379310346 201.7241379310345 M175.86206896551724 201.7241379310345 L217.24137931034483 201.7241379310345 M237.93103448275863 201.7241379310345 L300 201.7241379310345 M10.344827586206897 212.0689655172414 L20.689655172413794 212.0689655172414 M31.03448275862069 212.0689655172414 L51.724137931034484 212.0689655172414 M62.06896551724138 212.0689655172414 L124.13793103448276 212.0689655172414 M134.48275862068965 212.0689655172414 L155.17241379310346 212.0689655172414 M165.51724137931035 212.0689655172414 L258.62068965517244 212.0689655172414 M268.9655172413793 212.0689655172414 L279.3103448275862 212.0689655172414 M82.75862068965517 222.41379310344828 L103.44827586206897 222.41379310344828 M113.79310344827587 222.41379310344828 L124.13793103448276 222.41379310344828 M155.17241379310346 222.41379310344828 L186.20689655172413 222.41379310344828 M196.55172413793105 222.41379310344828 L217.24137931034483 222.41379310344828 M248.27586206896552 222.41379310344828 L258.62068965517244 222.41379310344828 M289.65517241379314 222.41379310344828 L300 222.41379310344828 M0 232.7586206896552 L72.41379310344828 232.7586206896552 M82.75862068965517 232.7586206896552 L103.44827586206897 232.7586206896552 M113.79310344827587 232.7586206896552 L134.48275862068965 232.7586206896552 M144.82758620689657 232.7586206896552 L165.51724137931035 232.7586206896552 M175.86206896551724 232.7586206896552 L217.24137931034483 232.7586206896552 M227.58620689655174 232.7586206896552 L237.93103448275863 232.7586206896552 M248.27586206896552 232.7586206896552 L268.9655172413793 232.7586206896552 M279.3103448275862 232.7586206896552 L289.65517241379314 232.7586206896552 M0 243.1034482758621 L10.344827586206897 243.1034482758621 M62.06896551724138 243.1034482758621 L72.41379310344828 243.1034482758621 M82.75862068965517 243.1034482758621 L103.44827586206897 243.1034482758621 M113.79310344827587 243.1034482758621 L124.13793103448276 243.1034482758621 M134.48275862068965 243.1034482758621 L165.51724137931035 243.1034482758621 M186.20689655172413 243.1034482758621 L196.55172413793105 243.1034482758621 M206.89655172413794 243.1034482758621 L217.24137931034483 243.1034482758621 M248.27586206896552 243.1034482758621 L258.62068965517244 243.1034482758621 M289.65517241379314 243.1034482758621 L300 243.1034482758621 M0 253.44827586206898 L10.344827586206897 253.44827586206898 M20.689655172413794 253.44827586206898 L51.724137931034484 253.44827586206898 M62.06896551724138 253.44827586206898 L72.41379310344828 253.44827586206898 M93.10344827586206 253.44827586206898 L113.79310344827587 253.44827586206898 M124.13793103448276 253.44827586206898 L134.48275862068965 253.44827586206898 M165.51724137931035 253.44827586206898 L175.86206896551724 253.44827586206898 M186.20689655172413 253.44827586206898 L196.55172413793105 253.44827586206898 M206.89655172413794 253.44827586206898 L258.62068965517244 253.44827586206898 M268.9655172413793 253.44827586206898 L279.3103448275862 253.44827586206898 M289.65517241379314 253.44827586206898 L300 253.44827586206898 M0 263.7931034482759 L10.344827586206897 263.7931034482759 M20.689655172413794 263.7931034482759 L51.724137931034484 263.7931034482759 M62.06896551724138 263.7931034482759 L72.41379310344828 263.7931034482759 M82.75862068965517 263.7931034482759 L124.13793103448276 263.7931034482759 M196.55172413793105 263.7931034482759 L217.24137931034483 263.7931034482759 M237.93103448275863 263.7931034482759 L248.27586206896552 263.7931034482759 M279.3103448275862 263.7931034482759 L289.65517241379314 263.7931034482759 M0 274.13793103448273 L10.344827586206897 274.13793103448273 M20.689655172413794 274.13793103448273 L51.724137931034484 274.13793103448273 M62.06896551724138 274.13793103448273 L72.41379310344828 274.13793103448273 M82.75862068965517 274.13793103448273 L103.44827586206897 274.13793103448273 M113.79310344827587 274.13793103448273 L144.82758620689657 274.13793103448273 M175.86206896551724 274.13793103448273 L196.55172413793105 274.13793103448273 M237.93103448275863 274.13793103448273 L248.27586206896552 274.13793103448273 M289.65517241379314 274.13793103448273 L300 274.13793103448273 M0 284.48275862068965 L10.344827586206897 284.48275862068965 M62.06896551724138 284.48275862068965 L72.41379310344828 284.48275862068965 M103.44827586206897 284.48275862068965 L144.82758620689657 284.48275862068965 M155.17241379310346 284.48275862068965 L165.51724137931035 284.48275862068965 M175.86206896551724 284.48275862068965 L227.58620689655174 284.48275862068965 M248.27586206896552 284.48275862068965 L268.9655172413793 284.48275862068965 M279.3103448275862 284.48275862068965 L289.65517241379314 284.48275862068965 M0 294.82758620689657 L72.41379310344828 294.82758620689657 M82.75862068965517 294.82758620689657 L113.79310344827587 294.82758620689657 M124.13793103448276 294.82758620689657 L155.17241379310346 294.82758620689657 M175.86206896551724 294.82758620689657 L217.24137931034483 294.82758620689657 M227.58620689655174 294.82758620689657 L237.93103448275863 294.82758620689657 M258.62068965517244 294.82758620689657 L268.9655172413793 294.82758620689657 M279.3103448275862 294.82758620689657 L289.65517241379314 294.82758620689657 "
          fill={
            Array [
              0,
              4278190080,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "stroke",
              "strokeWidth",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={
            Array [
              0,
              4278190080,
            ]
          }
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth={10.344827586206897}
          vectorEffect={0}
          x={0}
          y={0}
        />
      </RNSVGGroup>
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
